releasenotes/notes/fix-flare-genereation-deadlock-b40fca3c82469d40.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Fixed an issue where the datadog-agent status and datadog-agent flare commands would hang if a non-core Agent's port was occupied by a process that caused incoming connections to hang.
```

I think a simpler description would make it clearer, what do you think ? (we don't really need to give a detailed explanation of the bug, just an overview)
```suggestion
    Fixed an issue where the ``datadog-agent status`` and ``datadog-agent flare`` commands could hang while trying to reach other agent processes.
```